cpp/pybind/visualization/o3dvisualizer.cpp

```diff
@@ -344,6 +344,10 @@ void pybind_o3dvisualizer(py::module& m) {
                  "enable"_a)
             .def("show_skybox", &O3DVisualizer::ShowSkybox,
                  "Show/Hide the skybox", "show"_a)
+            .def("set_collapsable_vert_open",
```

Comment:
nit: consider naming `set_panel_open` instead for clarity.

Reply:
Updated in commit https://github.com/isl-org/Open3D/pull/6865/commits/a19cdf12bc6329a88047f2c602175005f7267eaf